### PR TITLE
Tohirun refactored

### PR DESCRIPTION
**Pull request by AbdullohIbnAlisher**
added some comments

### CODE REVIEW

File: WpfApp/MainWindow.xaml.cs [hunk 1]
```diff
@@ -24,6 +24,7 @@ public MainWindow()
         {
             InitializeComponent();
 //khbcbshksch
+//// added some comment
```
Comment (AbdullohIbnAlisher) [inline WpfApp/MainWindow.xaml.cs:27]: it is forbiddent

**Review by AbdullohIbnAlisher — COMMENTED** [review]
change code

File: WpfApp/MainWindow.xaml.cs (hunk 1 above)
Comment (TohirjonIbnAlisher) [reply WpfApp/MainWindow.xaml.cs:27]: ok you change

File: WpfApp/MainWindow.xaml.cs [hunk 2]
```diff
@@ -23,6 +23,8 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
+//khbcbshksch
+//// added some comment
```
Comment (AbdullohIbnAlisher) [inline WpfApp/MainWindow.xaml.cs:27]: it is forbidden

**Review by AbdullohIbnAlisher — COMMENTED** [review]
change code

**Comment by AbdullohIbnAlisher** [comment]
sjbcsbkchvcd

**Pull request closed without merging**